plugins/inputs/sqlserver/README.md

```diff
@@ -25,14 +27,36 @@ GO
 ```
 
 For Azure SQL Database, you require the View Database State permission and can create a user with a password directly in the database.
+
 ```sql
 CREATE USER [telegraf] WITH PASSWORD = N'mystrongpassword';
 GO
 GRANT VIEW DATABASE STATE TO [telegraf];
 GO
 ```
 
-### Configuration:
+For Azure SQL Elastic Pool, please follow the following instructions to collect metrics.
+
+On master logical database, create an SQL login 'telegraf' and assign it to the server-level role ##MS_ServerStateReader##. In addition, create also a user for this login.
+
+```sql
+CREATE LOGIN [telegraf] WITH PASSWORD = N'mystrongpassword';
+GO
+CREATE USER [telegraf] FOR LOGIN telegraf;
+GO
+ALTER SERVER ROLE ##MS_ServerStateReader##
+  ADD MEMBER telegraf
+GO
+```
+
+You can then collect metrics with this login by targetting any database attached to the pool. However, you must create the same user on this user database. Note: if you plan to add/remove databases from the pool, we recommend to create an empty database attached to the pool dedicated for this purpose.
+
```

Comment:
I agree. This change has been applied to the documentation.

Reply:
In this I am not getting the "anchor DB thing very clearly" I think we should state it clearly, aka if we recommended to have a separate small database for monitoring purposes or  ensure that we clearly say  to say use a SINGLE database within the pool to collect metrics as that isn't clear else there will be duplication of data.

Reply:
We could add "To avoid duplicate monitoring data, do not collect elastic pool metrics from more than one database in the same pool." I don't want to use "single" in this context because it clashes with our "single database" vs. "elastic pool database" terminology.

Reply:
Added this note into the README.md.